WORKSPACE

```diff
@@ -23,3 +25,40 @@ llvm_register_toolchains()
 load("//:setup_deps.bzl", "setup_dependencies")
 
 setup_dependencies()
+
+http_archive(
+    name = "rules_python",
+    sha256 = "29a801171f7ca190c543406f9894abf2d483c206e14d6acbd695623662320097",
+    strip_prefix = "rules_python-0.18.1",
+    url = "https://github.com/bazelbuild/rules_python/releases/download/0.18.1/rules_python-0.18.1.tar.gz",
+)
+
+load("@rules_python//python:repositories.bzl", "py_repositories", "python_register_toolchains")
+
+py_repositories()
+
+python_register_toolchains(
+    name = "python39",
+    # Available versions are listed in @rules_python//python:versions.bzl.
+    # We recommend using the same version your team is already standardized on.
+    python_version = "3.9",
+    # For now we're running the agents as root, we should fix this down the line.
```

Comment:
I've found it helpful to briefly comment what triggers or unblocks a to-do like this. Future editors can more readily evaluate, "OK, can/must we do this now?" instead of having to think through the implications of what blocks this/what depends on this.

Reply:
I've filed an issue here: https://github.com/sourcegraph/sourcegraph/issues/47943